config/secret_config.go

```diff
@@ -0,0 +1,99 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/jahvon/tuikit/types"
+	"gopkg.in/yaml.v3"
+)
+
+type SecretConfig struct {
```

Comment:
What do you think about merging this type into the `Secret` type defined in the vault package? https://github.com/jahvon/flow/blob/main/internal/vault/secret.go#L3

It will need to be some refactoring of existing references but that may be good approach for organizing relevant code together

Reply:
It should be possible. But vault.Secret looks like it represents only the secret value not the ref:value pair that secret_config is representing, so that might amount to a breaking change if the two were to be merged. I'll give it a try and report back.